cmd/internal/cli/pull.go

```diff
@@ -56,6 +59,9 @@ func init() {
 	PullCmd.Flags().StringVar(&PullLibraryURI, "library", "https://library.sylabs.io", "download images from the provided library")
 	PullCmd.Flags().SetAnnotation("library", "envkey", []string{"LIBRARY"})
 
+	PullCmd.Flags().StringVar(&PullPath, "path", "", "download images to the provided path")
+	PullCmd.Flags().SetAnnotation("path", "envkey", []string{"PULLPATH"})
```

Comment:
Lets make this annotation include `PULLFOLDER` so that this is compatible with previous use of the variable. So this last part would be `[]string{"PULLDIR", "PULLFOLDER"}`